lib/block-patterns/quote.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Quote.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Quote', 'Block pattern title', 'default' ),
+	'categories'  => array( 'text' ),
+	'content'     => '<!-- wp:group -->
+	<div class="wp-block-group"><div class="wp-block-group__inner-container"><!-- wp:separator {"className":"is-style-default"} -->
+	<hr class="wp-block-separator is-style-default"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:image {"align":"center","id":553,"width":150,"height":150,"sizeSlug":"large","linkDestination":"none","className":"is-style-rounded"} -->
+	<div class="wp-block-image is-style-rounded"><figure class="aligncenter size-large is-resized"><img src="https://s.w.org/images/core/5.8/portrait.jpg" alt="' . esc_html__( 'A side profile of a woman in a russet-colored turtleneck and white bag. She looks up with her eyes closed.', 'default' ) . '" class="wp-image-553" width="150" height="150"/></figure></div>
+	<!-- /wp:image -->
+	
+	<!-- wp:quote {"align":"center","className":"is-style-large"} -->
+	<blockquote class="wp-block-quote has-text-align-center is-style-large"><p>' . esc_html__( "\"Contributing makes me feel like I'm being useful to the planet.\"", 'default' ) . '</p><cite>' . esc_html__( '— Anna Wong, <em>Volunteer</em>', 'default' ) . '</cite></blockquote>
```

Comment:
This `<em>` tag is also getting escaped incorrectly: 

![Screen Shot 2021-04-05 at 9 12 01 AM](https://user-images.githubusercontent.com/1202812/113577483-558da280-95ef-11eb-9fd8-98b4817e2db8.png)

Reply:
I solved these using `wp_kses_post` (like TT1 does)